static/styles/zulip.css

```diff
@@ -852,6 +852,7 @@ td.pointer {
         font-weight: bold;
         color: hsl(0, 100%, 50%);
         position: relative;
+        transform: translateY(1px);
```

Comment:
Can we do this with simpler CSS than a `transform`?

Reply:
Sure, we can do. There were many options.